includes/templates/bootstrap/common/tpl_header.php

```diff
@@ -115,7 +115,6 @@
     </div>
 <?php } 
 // no HEADER_SALES_TEXT or SHOW_BANNERS_GROUP_SET2 ?>
-  </div>
```

Comment:
That's not an extra `</div>`; you (as I) were fooled by the fact that there's a `<div>` output with different parameters via `echo` around line 78.